src/test/java/edu/ucsb/cs156/courses/controllers/PSCourseControllerTests.java

```diff
@@ -447,6 +479,8 @@ public void api_courses_post__user_logged_in__secondary_enroll_code_adds_seconda
 	PSCourse expectedSecondary = PSCourse.builder().enrollCd("63388").psId(1L).user(u).id(0L).build();
         when(coursesRepository.save(eq(expectedSecondary))).thenReturn(expectedSecondary);
 
+        when(coursesRepository.findByPsIdAndEnrollCd(eq(1L), eq("63388"))).thenReturn(Optional.empty());
```

Comment:
I think the enroll code should be "63370" since that's the primary enroll code and what gets checked in the postCourses implementation.